l10n/Makefile

```
@@ -0,0 +1,147 @@
+				else \
+					echo "... waiting 10s to avoid API rate limits"; \
+					sleep 10; \
+				fi; \
+			} || exit 1; \
+		done; \
+		if [ -z "$$TO_TRANSLATE" ]; then \
+			echo "All languages are up to date. Nothing to translate."; \
+		fi; \
+	fi
+
+template:
```

checkmake (0.2.2)

[warning] Target body for "template" exceeds allowed length of 5 (8).

(maxbodylength)